lib/tbot/config/service_workload_identity_aws_ra.go

```diff
@@ -77,6 +77,19 @@ type WorkloadIdentityAWSRAService struct {
 	// defaults to 1 hour.
 	SessionRenewalInterval time.Duration `yaml:"session_renewal_interval"`
 
+	// CredentialProfileName is the name of the AWS credentials profile to
+	// write to. If unspecified, the profile will be named "default".
+	CredentialProfileName string `yaml:"credential_profile_name,omitempty"`
```

Comment:
I'm not too sure about this field name - it's a little long but I didn't want to confuse with the AWS Roles Anywhere "Profile" which is different.

Reply:
I like it! Clearer/more descriptive is always better IMO 👍🏻